starter-code/src/main.js

```diff
@@ -42,7 +42,13 @@ $(document).ready(function(){
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
+$('.back').click(function () {
+  var self = this;
+  if (self === $('.back')){
+    $(self).toggleClass('back front');
+  } else {
+  $(self).toggle('back');
```

Comment:
then you don't need all this stuff, you can flip the cards just with this line of code:
```
$('#card_fantastic four').children().toggleClass('back front');
```